src/module/iotjs_module_i2c.cpp

```diff
@@ -45,69 +48,207 @@ I2c* I2c::GetInstance() {
   return i2c;
 }
 
+
 JHANDLER_FUNCTION(SetAddress) {
-  IOTJS_ASSERT(!"Not implemented");
+  JHANDLER_CHECK(handler.GetArgLength() == 1);
+  JHANDLER_CHECK(handler.GetArg(0)->IsNumber());
+
+  I2c* i2c = I2c::GetInstance();
+  i2c->SetAddress(handler.GetArg(0)->GetNumber());
 
   handler.Return(JVal::Null());
```

Comment:
Is this intentional? To tell the truth I didn't checked, but I thought JS function usually return 'undefined' if there is no return value and everything went fine. What is the return value in Node.js?